ci/buildspec.yml

```diff
@@ -28,8 +28,9 @@ phases:
       # integration test is broken: https://github.com/awslabs/mxnet-model-server/issues/437
       #- python -m pytest mms/tests/integration_tests
       - pylint -rn --rcfile=./mms/tests/pylintrc mms/.
+      - python setup.py bdist_wheel --universal
```

Comment:
This wouldn't work in CI. Same version can't be uploaded to PyPi. I would suggest testing these changes on "test PyPi" before pushing PR.